tests/test_view.py

```diff
@@ -59,4 +60,156 @@ def test_pycon(self):
         self.assertFalse(view.mgr.pycon.python_redirect)
 
 
+def angle_axis(angle_deg, axis):
+    a = axis
+    angle = np.radians(angle_deg)
+
+    cos_angle = np.cos(angle)
+    sin_angle = np.sin(angle)
+
+    rotation = np.identity(3) * cos_angle
+
+    m = np.outer(a, a) * (1 - cos_angle)
+    n = np.array([
+        [0, -a[2], a[1]],
+        [a[2], 0, -a[0]],
+        [-a[1], a[0], 0]
+    ]) * sin_angle
+
+    rotation += m + n
+
+    return rotation
+
+
+@unittest.skip("GUI is not yet available for testing")
```

Comment:
Only skip tests when GUI is not build.  Test for `modmesh.HAS_VIEW`.  See https://github.com/solvcon/modmesh/blob/d92add6ee4f07caca20b52c943dd57b672e89a5a/tests/test_view.py#L45

Reply:
I've tried to do that one and I got an error: https://github.com/Tucchhaa/modmesh/actions/runs/10027785212/job/27713902533#step:14:2013

Seems that renderer can't be used in CI :/ At this point I don't know how to solve this problem, ahhh I hate testing so much 😣, but I will try to figure out what we can do with that

Reply:
Yes, `make run_viewer_pytest` works on my machine, but looks like it wouldn't work on CI

Reply:
Perhaps you can use [`GITHUB_ACTIONS`](https://docs.github.com/en/actions/learn-github-actions/variables) environment variable to turn it off temporarily.

Reply:
Yes.  Instead of always turning off the new camera tests, only turn them off when they are on GitHub Actions.

The environment-dependent behavior causes inconsistency and is hard to debug in the future. But it is better than having no test coverage for the new code.

Please leave a code comment stating a plan of improvement.  We will create an issue tracking it after merging the PR.